packages/rollup-contracts/contracts/L2ToL1MessageReceiver.sol

```diff
@@ -0,0 +1,58 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+/* Internal Imports */
+import {DataTypes as dt} from "./DataTypes.sol";
+
+contract L2ToL1MessageReceiver {
+    event L2ToL1MessageEnqueued(
+        address ovmSender,
+        bytes callData,
+        uint nonce
+    );
+
+    struct EnqueuedL2ToL1Message {
```

Comment:
If this will be passed between multiple contracts, should we put it in `DataTypes.sol`?

Reply:
As far as I can tell, it is only gonna show up here, so that's where I put it.